l1/l1.go

```diff
@@ -76,19 +76,25 @@ func (c *Client) subscribeToUpdates(ctx context.Context, updateChan chan *contra
 }
 
 func (c *Client) checkChainID(ctx context.Context) error {
-	gotChainID, err := c.l1.ChainID(ctx)
-	if err != nil {
-		return fmt.Errorf("retrieve Ethereum chain ID: %w", err)
-	}
+	for {
+		backOffDuration := time.Duration(1) * time.Second
+		gotChainID, err := c.l1.ChainID(ctx)
+		if err != nil {
+			c.log.Warnw("Failed to retrieve Ethereum chain ID: ", err)
```

Comment:
```suggestion
			c.log.Warnw("Failed to retrieve Ethereum chain ID, retrying", err)
```
Let's move this below the backoff calculation and add `"waitFor", backoffDuration` to the log so the user knows how long Juno will wait.